server/admin_management/api/app/routers/router_guards.py

```diff
@@ -236,13 +293,10 @@ def authorize_by_application_role(
     Check if role exists or not
     """
     if not role:
-        raise HTTPException(
-            status_code=HTTPStatus.BAD_REQUEST,
-            detail={
-                "code": ERROR_INVALID_ROLE_ID,
-                "description": "Requester has no appropriate role",
-            },
-            headers={"WWW-Authenticate": "Bearer"},
+        error_msg = "Requester has no appropriate role."
+        utils.raise_http_exception(
+            error_code=ERROR_INVALID_ROLE_ID,
+            error_msg=error_msg
         )
 
```

Comment:
just minor refactoring for using "utils.raise_http_exception" for several places below.